src/main/java/gov/cms/madie/models/cqm/datacriteria/AdverseEvent.java

```diff
@@ -6,19 +6,33 @@
 import gov.cms.madie.models.cqm.datacriteria.basetypes.Code;
 import lombok.AllArgsConstructor;
 import lombok.Data;
+import lombok.Getter;
 import lombok.NoArgsConstructor;
+import lombok.Setter;
 import lombok.experimental.SuperBuilder;
 
-import java.time.LocalDateTime;
+import java.time.ZonedDateTime;
 import java.util.List;
 
+import org.springframework.format.annotation.DateTimeFormat;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+
 @Data
 @SuperBuilder
 @NoArgsConstructor
 @AllArgsConstructor
+@Getter
+@Setter
 public class AdverseEvent extends DataElement {
-  private LocalDateTime authorDatetime;
-  private LocalDateTime relevantDatetime;
+  @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ss.SSSXXX")
+  @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME)
+  private ZonedDateTime authorDatetime;
+
+  @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ss.SSSXXX")
+  @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME)
+  private ZonedDateTime relevantDatetime;
+
   private Code severity;
   private FacilityLocation facilityLocation;
```

Comment:
this has Date field as well

Reply:
@adongare Thanks! Will get that done.

Reply:
@adongare This is done, with a few other ones that have FacilityLocation.